lib/logstash/outputs/elasticsearch/http_client_builder.rb

```diff
@@ -131,5 +149,10 @@ def self.setup_basic_auth(logger, params)
         :password => unsafe_escaped_password
       }
     end
+
+    private
+    def self.sanitize_url(url)
```

Comment:
Can we rename this to `dedupe_slashes` or something. Sanitization implies that it removes some sort of secret or dangerous threat. That's how its used in `SafeURI` which makes this name confusing here.

After that its LGTM, no need for another pass.